include/nfc/tnep/tag.h

```diff
@@ -172,29 +200,48 @@ int nfc_tnep_tag_tx_msg_buffer_register(uint8_t *tx_buff,
  *           Otherwise, a (negative) error code is returned.
  */
 int nfc_tnep_tag_init(struct k_poll_event *events, uint8_t event_cnt,
-		      struct nfc_ndef_msg_desc *msg,
 		      nfc_payload_set_t payload_set);
 
 /**
- * @brief Create the initial TNEP NDEF message.
+ * @brief Create the Initial TNEP NDEF message.
  *
- * This function create the Initial TNEP message. Initial NDEF message
- * has to contain at least one service parameters record. It can
- * contain also optional NDEF Records which can be used by NFC Poller
- * Device which does not support TNEP Protocol.
+ * This function creates the Initial TNEP message. Initial NDEF message
+ * has to contain at least one service parameters record defined
+ * using @ref NFC_TNEP_TAG_SERVICE_DEF. It can also contain optional NDEF
+ * records which can be used by NFC Poller Device which does not support
+ * TNEP Protocol.
  *
- * @param[in] svc Pointer to the first service information structure.
- * @param[in] svc_cnt Number of provided services for application.
- * @param[in] records Pointer to the first no TNEP NDEF Records structure.
- * @param[in] records_cnt Number of privided NDEF Records.
+ * @param[in] max_record_cnt Maximum count of the optional NDEF records
+ * @param[in] msg_encode_cb Callback function for encoding the
+ *                          Initial TNEP NDEF message.
```

Comment:
I see in examples that this parameter is optional and can be NULL. This should be clearly stated in parameter documentation.

Reply:
Ok. I added an additional description for this parameters.